src/js/form.js

```diff
@@ -11,7 +11,7 @@
       var id = child.get('id');
       var view = child.get('view');
       var options = child.get('options');
-      var defaultValue = this.model ? this.model.get(id) : '';
+      var defaultValue = options.defaultValue;
```

Comment:
@danielmurray great, thanks for taking a look! I'll update the code. For my own learning, how would a developer make use of the dialog in such a way that `this.model.get(id)` resulted in a defined value?